src/gl/index_buffer.js

```diff
@@ -23,10 +23,6 @@ class IndexBuffer {
 
         context.bindElementBuffer.set(this.buffer);
         gl.bufferData(gl.ELEMENT_ARRAY_BUFFER, array.arrayBuffer, this.dynamicDraw ? gl.DYNAMIC_DRAW : gl.STATIC_DRAW);
-
-        if (!this.dynamicDraw) {
-            delete array.arrayBuffer;
-        }
```

Comment:
It doesn't really make sense to remove the arrayBuffer from the `StructArray` because there are still views of the ArrayBuffer that we didn't remove here, so it couldn't be GCed anyway.